.globalconfig

```diff
@@ -1978,7 +1978,7 @@ dotnet_diagnostic.SA1410.severity = none
 
 # SA1411: Attribute constructor should not use unnecessary parenthesis
 # https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1411.md
-dotnet_diagnostic.SA1411.severity = none
+dotnet_diagnostic.SA1411.severity = warning
```

Comment:
Please move to .editorconfig

Reply:
We can discuss analyzer configuration in https://github.com/PowerShell/PowerShell/issues/24738.